tests/test_synthia_format.py

```diff
@@ -0,0 +1,150 @@
+
+from unittest import TestCase
+import os.path as osp
+
+import numpy as np
+
+from datumaro.components.annotation import (
+    AnnotationType, LabelCategories, Mask, MaskCategories,
+)
+from datumaro.components.dataset import Dataset
+from datumaro.components.environment import Environment
+from datumaro.components.extractor import DatasetItem
+from datumaro.util.test_utils import compare_datasets
+import datumaro.plugins.synthia_format as Synthia
+
+from .requirements import Requirements, mark_requirement
+
+DUMMY_INST_SEGM_DATASET_DIR = osp.join(osp.dirname(__file__),
+    'assets', 'synthia_dataset', 'synthia_dataset_1')
```

Comment:
It would be useful to give meaningful names to the subdirectories and not just 1, 2, 3. Ideally, they should be consistent with the names of the constants that refer to them.

Reply:
Fixed